server/Makefile

```diff
@@ -415,7 +415,7 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 endif
 endif
 
-check-style: golangci-lint plugin-checker vet ## Runs style/lint checks
+check-style: plugin-checker vet golangci-lint ## Runs style/lint checks
```

Comment:
Does the order matter?

Reply:
Yes, this was an intentional change to let vet run first, because if golangci fails, then it gives horrible output. Vet failures are much more understandable.